js/graviex.js

```diff
@@ -0,0 +1,991 @@
+            'symbol': symbol,
+            'type': sideType,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'remaining': remaining,
+            'status': status,
+            'trades': rawTrades,
+        }, market);
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrder() requires a order id argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+        };
+        if (id !== undefined) {
+            request['id'] = id;
+        }
+        const response = await this.privateGetOrder (this.extend (request, params));
+        // {
+        //   "id": 288336422,
+        //   "at": 1644217763,
+        //   "side": "sell",
+        //   "ord_type": "limit",
+        //   "price": "0.141",
+        //   "avg_price": "0.0",
+        //   "state": "wait",
+        //   "market": "ltcbtc",
+        //   "created_at": "2022-02-07T10:09:23+03:00",
+        //   "volume": "0.141",
+        //   "remaining_volume": "0.141",
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "trades": [],
+        //   "strategy": null
+        // }
+        return this.parseOrder (response);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+        };
+        let market = undefined;
+        if (symbol !== undefined) {
+            request['market'] = this.marketId (symbol);
+            market = this.market (symbol);
+        }
+        const response = await this.privateGetOrders (this.extend (request, params));
+        return this.parseOrders (response, market, since, limit);
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'desc': true,
+        };
+        let market = undefined;
+        const numericId = this.safeValue (params, 'market_id');
+        if (numericId !== undefined) {
+            request['market'] = numericId; // mutually exclusive with market_string
+        } else if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = market['id'];
+        }
+        const response = await this.privateGetTradesMy (this.extend (request, params));
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    async fetchTradesHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        const request = {
+            'order_by': 'desc',
+        };
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = this.marketId (symbol);
```

Comment:
again let's do `market['id']` here

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/162b106bdb42084917d6187a3a60fb6e0f55e7d1